### ISSUE TITLE
no resnet.eval() in encoder ?

### ISSUE THREAD

**Issue by curiousgradient** [HHTseng/video-classification#49]
@HHTseng Hi! Thanks for this repo. I noticed you did not do resnet.eval() in your encoder for frame feature extraction. Since we are freezing the weights of the resnet model, does it help to use the pretrained batch norm params while normalizing with current batch stats? 

https://github.com/HHTseng/video-classification/blob/99ebf204f0b1d737e38bc0d8b65aca128a57d7b1/ResNetCRNN/functions.py#L325-L341

**Comment by curiousgradient** [comment]
The parameters of batch norm are not updated. However, running stats will be updated.
